Content.Server/Genetics/Systems/GenomeSystem.cs

```diff
@@ -0,0 +1,121 @@
+using Content.Server.Genetics;
+using Content.Server.Genetics.Components;
+using Content.Shared.GameTicking;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Content.Server.Genetics.Systems;
+
+/// <summary>
+/// Assigns each <see cref="GenomePrototype"/> a random <see cref="GenomeLayout"/> roundstart.
+/// </summary>
+public sealed class GenomeSystem : EntitySystem
+{
+    [Dependency] private readonly IPrototypeManager _proto = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    // This is where all the genome layouts are stored.
+    // TODO: how would this be serialized for persistence???
```

Comment:
round entities eventually probably

Reply:
so is it good to just leave as a todo for whenever someone does that

Reply:
yeah its fine to leave